src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,28 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+type Props = {
+  error: ErrorTypes | null,
+};
+
+export const Notification: React.FC<Props> = ({ error }) => {
+  const [onButtonDelete, setOnButtonDelete] = useState(true);
+
+  return (
+    <div className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: onButtonDelete === false },
+    )}
+    >
+      <button
+        type="button"
+        className="delete"
+        aria-label="Mute volume"
```

Comment:
The 'aria-label' should describe the button action. Change the value to 'Close notification'.